embedded-pulsar/README.adoc

```diff
@@ -0,0 +1,34 @@
+=== embedded-pulsar
+
+==== Maven dependency
+
+.pom.xml
+[source,xml]
+----
+<dependency>
+    <groupId>com.playtika.testcontainers</groupId>
+    <artifactId>embedded-pulsar</artifactId>
+    <scope>test</scope>
+</dependency>
+----
+
+==== Consumes (via `bootstrap.properties`)
+
+* `embedded.pulsar.enabled` `(true|false, default is 'true')`
+* `embedded.pulsar.imageTag` `(default is '2.6.1')`
+** You can pick the desired version on https://hub.docker.com/r/apachepulsar/pulsar/tags[dockerhub]
+
+==== Produces
+
+* `embedded.pulsar.brokerUrl`
+* `embedded.pulsar.httpServiceUrl`
+
+==== Example
+
+Properties can be used to configure `PulsarClient` directly :
+
+----
+PulsarClient pulsarClient = PulsarClient.builder()
```

Comment:
any spring autoconfiguration for pulsar to have as an example hot to configure via properties?

Reply:
As far as I know there is no pulsar native support in spring boot so there is no autoconfiguration properties that cen be set with help of embedded ones.